### PR TITLE
FEATURE: BB-1907 Add MVF support for getValidElements API

### PR DESCRIPTION
**Pull request by qduval**
---

# PR checklist

- [ ] Change was tested using dev-release in Analytical Designer and Dashboards (if applicable).
- [ ] Change is described in [CHANGELOG.md](../blob/master/CHANGELOG.md).
- [ ] Migration guide (for major update) is written to [CHANGELOG.md](../blob/master/CHANGELOG.md).

### CODE REVIEW

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-js-oss-check/50/ : SUCCESS in 6s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-js-unit-tests-zuul-docker-el7/477/ : SUCCESS in 2m 17s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-js-oss-check/51/ : SUCCESS in 4s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-js-unit-tests-zuul-docker-el7/478/ : SUCCESS in 2m 09s

File: CHANGELOG.md [hunk 1]
```diff
@@ -8,12 +8,19 @@ The REST API versions in the table are just for your information as the values a
 
 |gooddata-js version | REST API version
 |:---:|:---:
+|\>= 13.0.0|5
 |\>= 12.0.0|4
 |\>= 10.0.0|3
 |<= 9.0.1|2
 
+<a name="13.0.0"></a>
```
Comment (martinmosko) [inline CHANGELOG.md]: this should be minor version as the change is backward compatible

File: CHANGELOG.md [hunk 2]
```diff
@@ -8,12 +8,19 @@ The REST API versions in the table are just for your information as the values a
...
+## 2020-01-21 Version [13.0.0](https://github.com/gooddata/gooddata-js/compare/v12.0.0...v13.0.0)
+
+- update REST API version to 5
+- update getValidElements to include measure value filters
```
Comment (martinmosko) [inline CHANGELOG.md]: `Added measure value filters support to getValidElements` ?

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-js-oss-check/52/ : SUCCESS in 4s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-js-unit-tests-zuul-docker-el7/479/ : SUCCESS in 1m 47s

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-js-merge-compliance-check/109/ : SUCCESS in 15s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-js-unit-tests-zuul-docker-el7/480/ : SUCCESS in 53s

**Pull request merged**